packages/react-migration-v8-v9/src/stories/ThemeShim/createFluent2ThemeForV8/DemoComponents/ThemeDemoPage.tsx

```diff
@@ -0,0 +1,134 @@
+import { CalloutDirectionalExample } from './FluentV8ComponentDemos/Callout.example';
+import { CheckboxExample } from './FluentV8ComponentDemos/Checkbox.example';
+import { ChoiceGroupExample } from './FluentV8ComponentDemos/Choicegroup.example';
+import { CommandBarBasicExample } from './FluentV8ComponentDemos/CommandBar.example';
+import { ContextualMenuDefaultExample } from './FluentV8ComponentDemos/ContextualMenu';
+import { DialogBasicExample } from './FluentV8ComponentDemos/Dialog.example';
+import { DropdownExample } from './FluentV8ComponentDemos/Dropdown.example';
+import { HoverCardInstantDismissExample } from './FluentV8ComponentDemos/Hovercard.example';
+import { MessageBarBasicExample } from './FluentV8ComponentDemos/MessageBar.example';
+import { PeoplePickerCompactExample } from './FluentV8ComponentDemos/PeoplePickerCompact.example';
+import { PeoplePickerListExample } from './FluentV8ComponentDemos/PeoplePickerList.example';
+import { PeoplePickerNormalExample } from './FluentV8ComponentDemos/people-picker.example';
+import { PivotBasicExample } from './FluentV8ComponentDemos/Pivot.example';
+import { SearchBoxFullSizeExample } from './FluentV8ComponentDemos/Searchbox.example';
+import { SliderBasicExample } from './FluentV8ComponentDemos/Slider.example';
+import { SpinButtonBasicExample } from './FluentV8ComponentDemos/SpinButton.example';
+import { SpinnerLabeledExample } from './FluentV8ComponentDemos/Spinner.example';
+import { SwatchColorPickerBasicExample } from './FluentV8ComponentDemos/SwatchColorPicker.example';
+import { TagPickerBasicExample } from './FluentV8ComponentDemos/TagPicker.example';
+import { TeachingBubbleBasicExample } from './FluentV8ComponentDemos/TeachingBubble.example';
+import { TextFieldBasicExample } from './FluentV8ComponentDemos/TextField.example';
+import { ToggleBasicExample } from './FluentV8ComponentDemos/Toggle.example';
+import { TooltipBasicExample } from './FluentV8ComponentDemos/Tooltip.example';
+
+import { Fluent2ForV8LightTheme } from '../../../../components/Theme/Fluent2ThemeForV8/fluent2-for-v8-light-theme';
+import { Fluent2ForV8DarkTheme } from '../../../../components/Theme/Fluent2ThemeForV8/fluent2-for-v8-dark-theme';
+import { Button, Menu, MenuItem, MenuList, MenuPopover, MenuTrigger } from '@fluentui/react-components';
+import { ChevronDown12Regular } from '@fluentui/react-icons';
+import { FluentComponentSamples } from './FluentV8ComponentSamples';
+
+const rootStyles = mergeStyles({ padding: 20 });
+
+export const Fluent2ForV8ThemeDemo: FunctionComponent<{}> = (props: React.PropsWithChildren<{}>) => {
+  const [selectedTheme, setSelectedTheme] = useState<ITheme>(Fluent2ForV8LightTheme);
+
+  const handLightSelection = () => {
+    setSelectedTheme(Fluent2ForV8LightTheme);
+  };
+
+  const handleDarkSelection = () => {
+    setSelectedTheme(Fluent2ForV8DarkTheme);
+  };
+
+  return (
+    <ThemeProvider theme={selectedTheme}>
+      <div className={rootStyles}>
```

Comment:
Would it be cool if I added a button to submit bugs here?

Reply:
Yeah, that's a good idea, for reference this is what it looks like on the main doc site, just a primary button that is a link.

![image](https://user-images.githubusercontent.com/40304143/208157330-f65185bd-2492-472b-8e2b-4bc8977e634f.png)